Install OpenCV

To install OpenCV, we need the following:

**Install dependencies**

```bash
sudo apt-get -y install libopencv-dev build-essential cmake git libgtk2.0-dev pkg-config python-dev python-numpy libdc1394-22 libdc1394-22-dev libjpeg-dev libpng12-dev libtiff4-dev libjasper-dev libavcodec-dev libavformat-dev libswscale-dev libxine-dev libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev libv4l-dev libtbb-dev libqt4-dev libfaac-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev libtheora-dev libvorbis-dev libxvidcore-dev x264 v4l-utils unzip
```

**Download OpenCV**

```bash
mkdir opencv
cd opencv
wget https://github.com/Itseez/opencv/archive/3.0.0-beta.zip -O opencv-3.0.0-beta.zip
unzip opencv-3.0.0-beta.zip
```

**Install OpenCV**

```bash
cd opencv-3.0.0-beta
mkdir build
cd build
cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D WITH_QT=ON -D WITH_OPENGL=ON ..
make -j $(nproc)
sudo make install
```

**System Update:** to get OpenCV working properly, we need to tell Ubuntu.

```bash
sudo /bin/bash -c 'echo "/usr/local/lib" > /etc/ld.so.conf.d/opencv.conf'
sudo ldconfig
```

- http://rodrigoberriel.com/2014/10/installing-opencv-3-0-0-on-ubuntu-14-04/

We are left with the following traceback error, when implementing `vagrant up`:

``` bash
...
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: /usr/bin/ld: ../../lib/libopencv_hal.a(mathfuncs.cpp.obj): relocation R_X86_64_32 against `_ZGVZN2cv9v_invsqrtERKNS_11v_float32x4EE4_1_5' can not be used when making a shared object; recompile with -fPIC
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: ../../lib/libopencv_hal.a: error adding symbols: Bad value
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: collect2: error: ld returned 1 exit status
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: make[2]: *** [lib/libopencv_core.so] Error 1
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: make[1]: *** [modules/core/CMakeFiles/opencv_core.dir/all] Error 2
==> default: Notice: /Stage[main]/Main/Exec[make-opencv]/returns: make: *** [all] Error 2
==> default: Error: /Stage[main]/Main/Exec[make-opencv]: Failed to call refresh: make returned 2 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[make-opencv]: make returned 2 instead of one of [0]
==> default: Notice: /Stage[main]/Main/Package[x264]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: /Stage[main]/Main/Package[checkinstall]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: /Stage[main]/Main/Package[python-numpy]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: /Stage[main]/Main/Package[libopencore-amrnb-dev]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: /Stage[main]/Main/Package[libtheora-dev]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: /Stage[main]/Main/Package[libfaac-dev]/ensure: ensure changed 'purged' to 'present'
==> default: Notice: Finished catalog run in 1323.62 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

We are left with the following traceback error, when implementing `vagrant up`:

``` bash
...
==> default: Notice: /Stage[main]/Main/File[/home/vagrant/opencv/release]/ensure: created
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error: The source directory "/home/vagrant/opencv" does not appear to contain CMakeLists.txt.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Specify --help for usage, or press the help button on the CMake GUI.
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: Failed to call refresh: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D INSTALL_C_EXAMPLES=ON -D INSTALL_PYTHON_EXAMPLES=ON -D BUILD_EXAMPLES=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D INSTALL_C_EXAMPLES=ON -D INSTALL_PYTHON_EXAMPLES=ON -D BUILD_EXAMPLES=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Notice: Finished catalog run in 418.56 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Before recursively copying the `cmake/` directory, we had a traceback similar to the following:

``` bash
...
==> default: Notice: /Stage[main]/Main/Exec[copy-cmake]: Triggered 'refresh' from 1 events
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:11 (include):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   include could not find load file:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     cmake/OpenCVMinDepVersions.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- The CXX compiler identification is GNU 4.8.2
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- The C compiler identification is GNU 4.8.2
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check for working CXX compiler: /usr/bin/c++
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check for working CXX compiler: /usr/bin/c++ -- works
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Detecting CXX compiler ABI info
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Detecting CXX compiler ABI info - done
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check for working C compiler: /usr/bin/cc
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check for working C compiler: /usr/bin/cc -- works
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Detecting C compiler ABI info
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Detecting C compiler ABI info - done
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:53 (include):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   include could not find load file:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     cmake/OpenCVUtils.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:55 (ocv_clear_vars):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Unknown CMake command "ocv_clear_vars".
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Configuring incomplete, errors occurred!
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: See also "/home/vagrant/opencv/release/CMakeFiles/CMakeOutput.log".
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: Failed to call refresh: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Notice: Finished catalog run in 487.54 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

However, after correctly implementing the recursive directory copy, we get the following traceback:

``` bash
...
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FSIGNED_CHAR - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_W
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_W - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_W
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_W - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WALL
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WALL - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WALL
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WALL - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_RETURN_TYPE
==> default: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_RETURN_TYPE - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_RETURN_TYPE
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_RETURN_TYPE - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_ADDRESS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_ADDRESS - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_ADDRESS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_ADDRESS - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_SEQUENCE_POINT
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_SEQUENCE_POINT - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WFORMAT
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WFORMAT - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WFORMAT
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WFORMAT - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_FORMAT_SECURITY
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WERROR_FORMAT_SECURITY - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WMISSING_DECLARATIONS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WMISSING_DECLARATIONS - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WMISSING_DECLARATIONS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WMISSING_DECLARATIONS - Success
==> default: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WMISSING_PROTOTYPES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WMISSING_PROTOTYPES - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WMISSING_PROTOTYPES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WMISSING_PROTOTYPES - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSTRICT_PROTOTYPES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSTRICT_PROTOTYPES - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WUNDEF
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WUNDEF - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WUNDEF
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WUNDEF - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WINIT_SELF
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WINIT_SELF - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WINIT_SELF
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WINIT_SELF - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WPOINTER_ARITH
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WPOINTER_ARITH - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WPOINTER_ARITH
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WPOINTER_ARITH - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSHADOW
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSHADOW - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSHADOW
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSHADOW - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSIGN_PROMO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WSIGN_PROMO - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSIGN_PROMO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WSIGN_PROMO - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_NARROWING
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_NARROWING - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_NARROWING
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_NARROWING - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_LONG_LONG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_WNO_LONG_LONG - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_LONG_LONG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_WNO_LONG_LONG - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_PTHREAD
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_PTHREAD - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_PTHREAD
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_PTHREAD - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FOMIT_FRAME_POINTER
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FOMIT_FRAME_POINTER - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FOMIT_FRAME_POINTER
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FOMIT_FRAME_POINTER - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE2
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE2 - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE2
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE2 - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE3
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_MSSE3 - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE3
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_MSSE3 - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FFUNCTION_SECTIONS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FFUNCTION_SECTIONS - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FFUNCTION_SECTIONS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FFUNCTION_SECTIONS - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FVISIBILITY_HIDDEN
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FVISIBILITY_HIDDEN - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FVISIBILITY_HIDDEN
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FVISIBILITY_HIDDEN - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FVISIBILITY_INLINES_HIDDEN
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_CXX_FVISIBILITY_INLINES_HIDDEN - Success
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FVISIBILITY_INLINES_HIDDEN
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Performing Test HAVE_C_FVISIBILITY_INLINES_HIDDEN - Failed
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for pthread.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for pthread.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check if the system is big endian
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Searching 16 bit integer
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for sys/types.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for sys/types.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for stdint.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for stdint.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for stddef.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for stddef.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check size of unsigned short
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check size of unsigned short - done
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Using unsigned short
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Check if the system is big endian - little endian
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found suitable version "1.2.8", minimum required is "1.2.3") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found TIFF: /usr/lib/x86_64-linux-gnu/libtiff.so (found version "4.0.3") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found JPEG: /usr/lib/x86_64-linux-gnu/libjpeg.so  
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at cmake/OpenCVFindLibsGrfmt.cmake:107 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "/home/vagrant/opencv/3rdparty/libwebp" which
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:451 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found Jasper: /usr/lib/x86_64-linux-gnu/libjasper.so (found version "1.900.1") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found PNG: /usr/lib/x86_64-linux-gnu/libpng.so (found version "1.2.50") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for /usr/include/libpng/png.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for /usr/include/libpng/png.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found OpenEXR: /usr/lib/x86_64-linux-gnu/libIlmImf.so
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVFindLibsGUI.cmake:18 (find_package):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   By not providing "FindQt5Core.cmake" in CMAKE_MODULE_PATH this project has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   asked CMake to find a package configuration file provided by "Qt5Core", but
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMake did not find one.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Could not find a package configuration file provided by "Qt5Core" with any
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   of the following names:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     Qt5CoreConfig.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     qt5core-config.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Add the installation prefix of "Qt5Core" to CMAKE_PREFIX_PATH or set
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Core_DIR" to a directory containing one of the above files.  If
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Core" provides a separate development package or SDK, be sure it has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   been installed.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:452 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVFindLibsGUI.cmake:19 (find_package):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   By not providing "FindQt5Gui.cmake" in CMAKE_MODULE_PATH this project has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   asked CMake to find a package configuration file provided by "Qt5Gui", but
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMake did not find one.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Could not find a package configuration file provided by "Qt5Gui" with any
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   of the following names:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     Qt5GuiConfig.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     qt5gui-config.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Add the installation prefix of "Qt5Gui" to CMAKE_PREFIX_PATH or set
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Gui_DIR" to a directory containing one of the above files.  If "Qt5Gui"
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   provides a separate development package or SDK, be sure it has been
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   installed.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:452 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVFindLibsGUI.cmake:20 (find_package):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   By not providing "FindQt5Widgets.cmake" in CMAKE_MODULE_PATH this project
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   has asked CMake to find a package configuration file provided by
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Widgets", but CMake did not find one.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Could not find a package configuration file provided by "Qt5Widgets" with
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   any of the following names:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     Qt5WidgetsConfig.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     qt5widgets-config.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Add the installation prefix of "Qt5Widgets" to CMAKE_PREFIX_PATH or set
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Widgets_DIR" to a directory containing one of the above files.  If
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Widgets" provides a separate development package or SDK, be sure it has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   been installed.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:452 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVFindLibsGUI.cmake:21 (find_package):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   By not providing "FindQt5Test.cmake" in CMAKE_MODULE_PATH this project has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   asked CMake to find a package configuration file provided by "Qt5Test", but
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMake did not find one.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Could not find a package configuration file provided by "Qt5Test" with any
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   of the following names:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     Qt5TestConfig.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     qt5test-config.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Add the installation prefix of "Qt5Test" to CMAKE_PREFIX_PATH or set
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Test_DIR" to a directory containing one of the above files.  If
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Test" provides a separate development package or SDK, be sure it has
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   been installed.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:452 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVFindLibsGUI.cmake:22 (find_package):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   By not providing "FindQt5Concurrent.cmake" in CMAKE_MODULE_PATH this
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   project has asked CMake to find a package configuration file provided by
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Concurrent", but CMake did not find one.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Could not find a package configuration file provided by "Qt5Concurrent"
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   with any of the following names:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     Qt5ConcurrentConfig.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     qt5concurrent-config.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Add the installation prefix of "Qt5Concurrent" to CMAKE_PREFIX_PATH or set
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Concurrent_DIR" to a directory containing one of the above files.  If
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   "Qt5Concurrent" provides a separate development package or SDK, be sure it
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   has been installed.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:452 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_X11
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_X11 - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_WIN
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_WIN - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_QWS
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_QWS - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_MAC
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for Q_WS_MAC - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found Qt4: /usr/bin/qmake (found version "4.8.6") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for gethostbyname
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for gethostbyname - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for connect
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for connect - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for remove
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for remove - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for shmat
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for shmat - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for IceConnectionNumber in ICE
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for IceConnectionNumber in ICE - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found X11: /usr/lib/x86_64-linux-gnu/libX11.so
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-base-1.0'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   package 'gstreamer-base-1.0' not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-video-1.0'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   package 'gstreamer-video-1.0' not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-app-1.0'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   package 'gstreamer-app-1.0' not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-riff-1.0'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   package 'gstreamer-riff-1.0' not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-pbutils-1.0'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   package 'gstreamer-pbutils-1.0' not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-base-0.10'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found gstreamer-base-0.10, version 0.10.36
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-video-0.10'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found gstreamer-video-0.10, version 0.10.36
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-app-0.10'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found gstreamer-app-0.10, version 0.10.36
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-riff-0.10'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found gstreamer-riff-0.10, version 0.10.36
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'gstreamer-pbutils-0.10'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found gstreamer-pbutils-0.10, version 0.10.36
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libdc1394-2'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libdc1394-2, version 2.2.1
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libv4l1'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libv4l1, version 1.0.1
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for linux/videodev.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for linux/videodev.h - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for linux/videodev2.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for linux/videodev2.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for sys/videoio.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for sys/videoio.h - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libavcodec'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libavcodec, version 54.35.0
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libavformat'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libavformat, version 54.20.4
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libavutil'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libavutil, version 52.3.0
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'libswscale'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found libswscale, version 2.1.1
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for libavformat/avformat.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for libavformat/avformat.h - found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for ffmpeg/avformat.h
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Looking for ffmpeg/avformat.h - not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- checking for module 'tbb'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   found tbb, version 4.2~20130725
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at cmake/OpenCVFindIPP.cmake:212 (include):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   include could not find load file:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:     /home/vagrant/opencv/3rdparty/ippicv/downloader.cmake
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   cmake/OpenCVFindLibsPerf.cmake:12 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:454 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found PythonInterp: /usr/bin/python2.7 (found suitable version "2.7.6", minimum required is "2.7") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython2.7.so (found suitable exact version "2.7.6") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Found PythonInterp: /usr/bin/python3.4 (found suitable version "3.4", minimum required is "3.4") 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Could NOT find PythonLibs (missing:  PYTHON_LIBRARIES PYTHON_INCLUDE_DIRS) (Required is exact version "3.4.0")
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Traceback (most recent call last):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   File "<string>", line 1, in <module>
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: ImportError: No module named 'numpy'
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Could NOT find JNI (missing:  JAVA_AWT_LIBRARY JAVA_JVM_LIBRARY JAVA_INCLUDE_PATH JAVA_INCLUDE_PATH2 JAVA_AWT_INCLUDE_PATH) 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Warning at cmake/OpenCVDetectOpenCL.cmake:22 (message):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   Can't use OpenCL
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:493 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Could NOT find Matlab (missing:  MATLAB_MEX_SCRIPT MATLAB_INCLUDE_DIRS MATLAB_ROOT_DIR MATLAB_LIBRARIES MATLAB_LIBRARY_DIRS MATLAB_MEXEXT MATLAB_ARCH MATLAB_BIN) 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- VTK is not found. Please set -DVTK_DIR in CMake to VTK build directory, or to VTK install subdirectory with VTKConfig.cmake file
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:535 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "include" which is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:538 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "modules" which is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:541 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "doc" which is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:544 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "data" which is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at CMakeLists.txt:548 (add_subdirectory):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   add_subdirectory given source "apps" which is not an existing directory.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- General configuration for OpenCV .. =====================================
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Version control:               unknown
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Platform:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Host:                        Linux 3.13.0-49-generic x86_64
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     CMake:                       2.8.12.2
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     CMake generator:             Unix Makefiles
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     CMake build tool:            /usr/bin/make
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Configuration:               RELEASE
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   C/C++:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Built as dynamic libs?:      YES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C++ Compiler:                /usr/bin/c++  (ver 4.8.2)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C++ flags (Release):         -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wno-narrowing -Wno-delete-non-virtual-dtor -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -msse3 -ffunction-sections -fvisibility=hidden -fvisibility-inlines-hidden -O3 -DNDEBUG  -DNDEBUG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C++ flags (Debug):           -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wno-narrowing -Wno-delete-non-virtual-dtor -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -msse3 -ffunction-sections -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C Compiler:                  /usr/bin/cc
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C flags (Release):           -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wno-narrowing -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -msse3 -ffunction-sections -fvisibility=hidden -O3 -DNDEBUG  -DNDEBUG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C flags (Debug):             -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wno-narrowing -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -msse3 -ffunction-sections -fvisibility=hidden -g  -O0 -DDEBUG -D_DEBUG
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Linker flags (Release):      
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Linker flags (Debug):        
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Precompiled headers:         YES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   OpenCV modules:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     To be built:                 -
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Disabled:                    -
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Disabled by dependency:      -
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Unavailable:                 -
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   GUI: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     QT 4.x:                      YES (ver 4.8.6 EDITION = OpenSource)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     QT OpenGL support:           YES (/usr/lib/x86_64-linux-gnu/libQtOpenGL.so)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     OpenGL support:              YES (/usr/lib/x86_64-linux-gnu/libGLU.so /usr/lib/x86_64-linux-gnu/libGL.so /usr/lib/x86_64-linux-gnu/libSM.so /usr/lib/x86_64-linux-gnu/libICE.so /usr/lib/x86_64-linux-gnu/libX11.so /usr/lib/x86_64-linux-gnu/libXext.so)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     VTK support:                 NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Media I/O: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     ZLib:                        /usr/lib/x86_64-linux-gnu/libz.so (ver 1.2.8)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     JPEG:                        /usr/lib/x86_64-linux-gnu/libjpeg.so (ver )
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     WEBP:                        build (ver )
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     PNG:                         /usr/lib/x86_64-linux-gnu/libpng.so (ver 1.2.50)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     TIFF:                        /usr/lib/x86_64-linux-gnu/libtiff.so (ver 42 - 4.0.3)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     JPEG 2000:                   /usr/lib/x86_64-linux-gnu/libjasper.so (ver 1.900.1)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     OpenEXR:                     /usr/lib/x86_64-linux-gnu/libImath.so /usr/lib/x86_64-linux-gnu/libIlmImf.so /usr/lib/x86_64-linux-gnu/libIex.so /usr/lib/x86_64-linux-gnu/libHalf.so /usr/lib/x86_64-linux-gnu/libIlmThread.so (ver 1.6.1)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     GDAL:                        NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Video I/O:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     DC1394 1.x:                  NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     DC1394 2.x:                  YES (ver 2.2.1)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     FFMPEG:                      YES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       codec:                     YES (ver 54.35.0)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       format:                    YES (ver 54.20.4)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       util:                      YES (ver 52.3.0)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       swscale:                   YES (ver 2.1.1)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       gentoo-style:              YES
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     GStreamer:                   
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       base:                      YES (ver 0.10.36)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       video:                     YES (ver 0.10.36)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       app:                       YES (ver 0.10.36)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       riff:                      YES (ver 0.10.36)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --       pbutils:                   YES (ver 0.10.36)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     OpenNI:                      NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     OpenNI PrimeSensor Modules:  NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     OpenNI2:                     NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     PvAPI:                       NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     GigEVisionSDK:               NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     UniCap:                      NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     UniCap ucil:                 NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     V4L/V4L2:                    Using libv4l (ver 1.0.1)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     XIMEA:                       NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Xine:                        NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Other third-party libraries:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use IPP:                     IPP not found
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use IPP Async:               NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use Eigen:                   NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use TBB:                     YES (ver 4.2 interface 7000)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use OpenMP:                  NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use GCD                      NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use Concurrency              NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use C=:                      NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use Cuda:                    NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Use OpenCL:                  NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Python 2:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Interpreter:                 /usr/bin/python2.7 (ver 2.7.6)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Python 3:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Interpreter:                 /usr/bin/python3.4 (ver 3.4)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Python (for build):            /usr/bin/python2.7
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Java:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     ant:                         NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     JNI:                         NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Java wrappers:               NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Java tests:                  NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Matlab:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     mex:                         NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Documentation:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Build Documentation:         NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Sphinx:                      NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     PdfLaTeX compiler:           NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     PlantUML:                    NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Doxygen:                     NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Tests and samples:
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Tests:                       NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     Performance tests:           NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --     C/C++ Examples:              NO
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   Install path:                  /usr/local
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: --   cvconfig.h is in:              /home/vagrant/opencv/release
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- -----------------------------------------------------------------
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: CMake Error at /usr/share/cmake-2.8/Modules/CPack.cmake:395 (message):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CPack license resource file: "/home/vagrant/opencv/LICENSE" could not be
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   found.
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: Call Stack (most recent call first):
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   /usr/share/cmake-2.8/Modules/CPack.cmake:400 (cpack_check_file_exists)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   cmake/OpenCVPackaging.cmake:108 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns:   CMakeLists.txt:1110 (include)
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: 
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: -- Configuring incomplete, errors occurred!
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: See also "/home/vagrant/opencv/release/CMakeFiles/CMakeOutput.log".
==> default: Notice: /Stage[main]/Main/Exec[cmake-opencv]/returns: See also "/home/vagrant/opencv/release/CMakeFiles/CMakeError.log".
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: Failed to call refresh: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[cmake-opencv]: cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D WITH_QT=ON -D WITH_OPENGL=ON .. returned 1 instead of one of [0]
==> default: Notice: Finished catalog run in 844.91 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

--- a/manifests/default.pp
+++ b/manifests/default.pp
@@ -140,4 +140,5 @@ exec {'update-opencv':
 exec {'ldconfig':
     command     => 'ldconfig',
     refreshonly => true,
+    path        => '/sbin/',
 }

